### PR TITLE
openflow: bugfix - race with ovs-ofctl and functional tests.

### LINKED ISSUE #370: ovs_of: functional tests are needed

### PR DESCRIPTION
**Pull request by pierrecregut**
Race between ovs-ofctl create and ovs-ofctl monitor when the bridge appears in ovsdb database

- added a functional test on openflow probe. It may exhibit the
problem
- added info logging (as for other probes) to make functional
  tests log usable.

closes #370

### CODE REVIEW

File: topology/probes/ovs_of.go [hunk 1]
```diff
@@ -292,29 +292,35 @@ func launchOnSwitch(cmd []string) (string, error) {
...
-	reader := bufio.NewReader(out)
-	var line string
 
 	go func() {
-		for {
-			line, err = reader.ReadString('\n')
-			if err == io.EOF {
+		for ctx.Err() == nil {
+			out, err := executor.ExecCommandPipe(ctx, cmd[0], cmd[1:]...)
+			if err != nil {
```
Comment (nplanel) [inline topology/probes/ovs_of.go]: log the error here like
logging.GetLogger().Errorf("Can't execute command %v", cmd)

File: topology/probes/ovs_of.go [hunk 2]
```diff
@@ -494,6 +502,7 @@ func (o *OvsOfProbe) OnOvsBridgeAdd(bridgeNode *graph.Node) {
 		return
 	}
 	bridgeProbe, err := o.NewBridgeProbe(hostName, bridgeName, uuid, bridgeNode)
+	logging.GetLogger().Infof("Probe added for %s on %s", bridgeName, hostName)
```
Comment (nplanel) [inline topology/probes/ovs_of.go]: log the uuid as well

File: topology/probes/ovs_of.go (hunk 1 above)
Comment (pierrecregut) [reply topology/probes/ovs_of.go]: Yes missing logging.

File: topology/probes/ovs_of.go (hunk 2 above)
Comment (pierrecregut) [reply topology/probes/ovs_of.go]: Yes and made more consistent with error case.

**Pull request merged**